docs/resources/app_v2.md

```diff
@@ -21,6 +21,23 @@ resource "rancher2_app_v2" "foo" {
 }
 ```
 
+### Create an App from a Helm Chart using a different registry
+
+The `system_default_registry` argument can override the global value at App installation time. If  argument is not provided, the global value for System Default Registry will be used instead.
```

Comment:
Suggestion: `at App installation.` and remove extra space between If/argument

Reply:
Keen eyes